cylc/flow/graph_parser.py

```diff
@@ -328,16 +331,39 @@ def parse_graph(self, graph_string: str) -> None:
                 next_line = non_blank_lines[i + 1]
             except IndexError:
                 next_line = ''
-                if this_line.endswith(self.__class__.ARROW):
-                    # Last line can't end with an arrow.
-                    raise GraphParseError(
-                        f"Trailing arrow: {this_line}")
+                for seq in self.CONTINUATION_STRS:
+                    if this_line.endswith(seq):
+                        # Last line can't end with an arrow, & or |.
+                        raise GraphParseError(
+                            f"Trailing continuation sequence ({seq}):"
```

Comment:
```suggestion
                            f"Dangling ({seq}):"
```
"Trailing" is OK, so long as there's a subsequent line that fits.

Also I'm don't think `sequence` is a great name for these things in the user-facing validation message. Probably sufficient to just print what's dangling without naming it.